ros_buildfarm/templates/prerelease/prerelease_script_check.sh.em

```diff
@@ -2,12 +2,12 @@ set +e
 _ls_prerelease_scripts=`ls | grep -v "prerelease.*\.sh"`
 set -e
 if [ "$_ls_prerelease_scripts" != "" ]; then
-  echo "This script should be executed either in an empty folder or in a folder only containing the prerelease scripts" 1>&2
+  echo "Files from previous prerelease tests are found. These files help speeding things up if you mean to repeat the test for the same repository. If you're testing different repository than the one in previous test, you need to remove them as this script should be executed either in an empty folder or in a folder only containing the prerelease scripts" 1>&2
```

Comment:
This makes the assumption that the folder does contain "files from previous prerelease tests". But that doesn't have to be the case. The user might have just called the script the first time but passed a non-empty folder. So I think this needs to be rephrased more carefully.